### PR TITLE
add required param site_slug

### PR DESCRIPTION
**Pull request by vitalikdanchenko**
## Demo
http://void-docs.talkable.com/

### CODE REVIEW

File: source/api_v2/visitors.rst [hunk 1]
```diff
@@ -17,6 +17,8 @@ This API allows you to create visitors. See examples below.
    ================= ========================================================
    Parameter         Description
    ================= ========================================================
+   site_slug         Your Talkable Site ID. You can get this from your
```
Comment (zhuravel) [inline source/api_v2/visitors.rst]: Example below should also be updated.
Let’s also convert it to curl example as we have for other endpoints.
```
Create a Visitor
................

.. code-block:: bash

   curl -H "Content-Type: application/json" \
        -X POST \
        -d '{"api_key":"i9uil7nQgDjucCiTJu","site_slug":"my-store","data":{"uuid":"b3967d36-4e7f-46bc-92b3-57344347cd6a"}}' \
        https://www.talkable.com/api/v2/visitors
```

**Comment by DanielKehoe** [comment]
[Elsewhere])(https://github.com/talkable/talkable-ruby/issues/22) we discussed avoiding the term "site slug" and using the term "site id" instead. For consistency. Thoughts?

**Comment by vitalikdanchenko** [comment]
I also would like to have it consistent everywhere, but I agree with @zhuravel. Since we use suffix `_slug` a lot in public API (`site_slug`, `campaign_slug`, `origin_slug`, etc.) I would proceed with it. Also I prefer `slug` more because it's broader and handle more options that just `id`.

**Comment by DanielKehoe** [comment]
Good to have this discussion. Thanks.

**Comment by zhuravel** [comment]
@vitalikdanchenko Any update on this?

**Comment by vitalikdanchenko** [comment]
@zhuravel updated, sorry I lost this pull request

**Pull request merged**